ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/package.scala

```diff
@@ -8,18 +8,18 @@ import com.digitalasset.daml.lf.data.Ref.LedgerString
 package object ledger {
 
   /** Identifiers used to correlate submission with results. */
+  type CommandId = LedgerString
   val CommandId: LedgerString.type = LedgerString
-  type CommandId = CommandId.T
 
   /** Identifiers used for correlating submission with a workflow.  */
+  type WorkflowId = LedgerString
```

Comment:
This still allows using a value of type `CommandId` for a parameter of type `WorkflowId`, right?

Reply:
Yes those live outside the engine, we have to handle separately. 
I would simply tag those.